src/material-experimental/mdc-menu/menu.spec.ts

```diff
@@ -762,6 +762,44 @@ describe('MatMenu', () => {
     flush();
   }));
 
+  it('should respect the DOM order, rather than insertion order, when moving focus using ' +
```

Comment:
I ported this test over, because it got in through a very old PR that got in recently which was opened before the MDC menu existed.